src/netex-convertor/period-tickets/periodTicketNetexTemplate.xml

```diff
@@ -541,7 +541,7 @@
 							<DistributionChannelType>online</DistributionChannelType>
 							<IsObligatory>false</IsObligatory>
 						</DistributionChannel>
-						<DistributionChannel id="fxc:at_stop" version="fxc:v1.0">
+						<DistributionChannel id="fxc:atStop" version="fxc:v1.0">
```

Comment:
Sorry if I misunderstood what you were wanting to do with making this camelcase the other day but I'm slightly hesitant to make these changes to the shared reference data, NK wanted us to let him know about any changes we were making to it and this introduces some inconsistency around casing which I'm not sure he'd be happy with

Reply:
It might be better just to map the keys we get from the matching json to the ones that are expected by these distribution channels

Reply:
I think for now we can go with this but I might raise a separate ticket to compare it to the original and see what's changed and out of that, what actually needs to change. Then we can report back to NK about it.